integrations/docker/build-all.sh

```diff
@@ -25,19 +25,33 @@ set -e
 
 SUB_PATH="*"
 ALL_ARGS=""
+SKIP_BUILT=0
 
 for i in "$@"; do
     case $i in
+    -h | --help)
+       echo "Supported arguments:"
+       echo "  -h/--help       This help text"
+       echo "  -p/--path       TODO: document this"
```

Comment:
i like docs :)

Reply:
I wish I had an idea what -p/--path actually does, but I could not figure it out at a quick glance. I just wanted at least the thing that I added documented :)

Reply:
I will try to figure out what it does ... It was a miss from our side not documenting things to start.